src/test/scala/modbat/mbt/ModbatTestHarness.scala

```diff
@@ -20,12 +20,38 @@ import modbat.log.Log
 import modbat.util.CloneableRandom
 
 object ModbatTestHarness {
+  def getNamesofFile(args: Array[String], td: org.scalatest.TestData): (String, String, String) = {
+    // Be careful, this function create folder if doesn't exist
+    var name_output = "log/modbat/"
+    var arguments_name = ""
+    for ( x <- args ) {
+      if (x contains "modbat"){
```

Comment:
Perhaps this should be "modbat.", as part of a package name, so the substring match is more specific?